egs2/TEMPLATE/asr1/db.sh

```diff
@@ -71,6 +71,7 @@ LJSPEECH=downloads
 MUSAN=
```

Comment:
A final note, could you also add an entry in egs2/README.md for nit_song?

Reply:
Done